sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs

```diff
@@ -30,15 +32,17 @@ public async Task SendingAnSmsMessage()
             try
             {
                 #region Snippet:Azure_Communication_Sms_Tests_SendAsync
-                SendSmsResponse result = await client.SendAsync(
-                   //@@ from: new PhoneNumber("+18001230000"), // Phone number acquired on your Azure Communication resource
-                   //@@ to: new PhoneNumber("+18005670000"),
-                   /*@@*/ from: new PhoneNumberIdentifier(TestEnvironment.PhoneNumber),
-                   /*@@*/ to: new PhoneNumberIdentifier(TestEnvironment.PhoneNumber),
+                SmsSendResult result = await client.SendAsync(
+                   //@@ from: "+18001230000" // Phone number acquired on your Azure Communication resource
+                   //@@ to: "+18005670000",
+                   /*@@*/ from: TestEnvironment.ToPhoneNumber,
```

Comment:
`from` is using the `To` variable and vice versa. Same for the next test

Reply:
Good catch! thanks